insights/parsers/yumlog.py

```diff
@@ -117,7 +117,7 @@ def parse_content(self, content):
                 timestamp = ' '.join([month, day, time])
                 state = state.rstrip(':')
                 pkg = pkg.split(':')[-1].strip()
-                if state == self.ERASED:
+                if state == self.ERASED and len(pkg.split(".")) == 1:
```

Comment:
How about
`if state == self.ERASED and "." in pkg`

Reply:
it should be` if state == self.ERASED and "." not in pkg`, updated, please have a check.